api-reference/reference.md

```diff
@@ -1563,6 +1563,167 @@ The intended purpose of the file. Currently, only `batch` is supported.
 
 ---
 
+## Embeddings
+
+### Create embeddings
+
+`POST https://api.kluster.ai/v1/embeddings`
+
+Creates an embedding vector representing the input text. Machine learning models and algorithms can easily consume these vector representations to understand semantic relationships between pieces of text.
+
+<div class="grid" markdown>
+<div markdown>
+
+**Request**
+
+`input` ++"string or array"++ <span class="required" markdown>++"required"++</span>
+
+Input text to embed, encoded as a string or array of tokens. To embed multiple inputs in a single request, pass an array of strings or an array of token arrays. The input must not exceed the max input tokens for the model (8192 tokens for `BAAI/bge-m3`), it cannot be an empty string, and any array must be 2048 dimensions or less.
+
+---
+
+`model` ++"string"++ <span class="required" markdown>++"required"++</span>
+
+ID of the model to use.
+
+---
+
+`encoding_format` ++"string"++
+
+The format to return the embeddings in. Can be either `float` or `base64`. Defaults to `float`.
+
+---
+
+Please note, the embeddings endpoint doesn’t support the dimensions parameter. Models such as `BAAI/bge‑m3` always return a fixed 1024‑dimensional vector. Supplying dimensions will trigger a `400 Bad Request`.
```

Comment:
```suggestion
`dimensions` <span class="not-supported" markdown>++"not supported"++</span>

Please note, the embeddings endpoint doesn’t support the `dimensions` parameter. Models such as `BAAI/bge‑m3` always return a fixed 1024‑dimensional vector. Supplying `dimensions` will trigger a `400 Bad Request`.
```

I feel like this is getting lost a little bit. What do you think about doing something like this?

Reply:
Looks great!